src/__tests__/fireEvent.test.tsx

```diff
@@ -56,6 +56,28 @@ const CustomEventComponentWithCustomName = ({
 );
 
 describe('fireEvent', () => {
+  test('QUESTION: is there a kind of event that takes multiple arguments?', () => {
```

Comment:
AFAIK events by convention take a single argument of `Event` type or subtype. We also have `onChangeText` which gets a single `string` value, but that seems to be simplified version of [`onChange`](https://reactnative.dev/docs/textinput#onchange). I would assume that we can focus only on handling single argument. That should simplify the typing, while we can always refactor if there would be any such event.

Reply:
Thanks for this info! BTW I noticed that I didn't clearly link to our additional questions from here. Here they are: https://github.com/callstack/react-native-testing-library/issues/714#issuecomment-1278841845